unittest/vslib/TestMACsecFilterStateGuard.cpp

```diff
@@ -0,0 +1,18 @@
+#include "MACsecFilterStateGuard.h"
+
+#include <gtest/gtest.h>
+
+using namespace saivs;
+
```

Comment:
you are missing update Makefile.am for tests

Reply:
Done